ambry-api/src/main/java/com/github/ambry/server/AccountStatsStore.java

```diff
@@ -26,29 +27,27 @@
    * and container. This method will be used for individual ambry server to store local stats.
    * @param statsWrapper The {@link StatsWrapper} that contains stats and other metadata.
    */
-  void storeStats(StatsWrapper statsWrapper) throws Exception;
+  void storeAccountStats(StatsWrapper statsWrapper) throws Exception;
 
   /**
    * Store aggregated stats in the {@link StatsSnapshot}. The StatsSnapshot should include account and container. This
    * method will be used for aggregation task to store aggregated stats.
    * @param snapshot The {@link StatsSnapshot} that contains aggregated container usage.
    * @throws Exception
    */
-  void storeAggregatedStats(StatsSnapshot snapshot) throws Exception;
+  void storeAggregatedAccountStats(StatsSnapshot snapshot) throws Exception;
 
   /**
-   * Return individual ambry server's stats for the given {@code clusterName} and {@code hostname}. This is the stats
-   * stored by method {@link #storeStats}.
-   * @param clusterName The clusterName.
+   * Return individual ambry server's stats for the given {@code hostname}. This is the stats stored by method {@link #storeAccountStats}.
    * @param hostname The hostname.
-   * @return {@link StatsWrapper} of given {@code hostname} in the {@code clusterName}.
+   * @return {@link StatsWrapper} of given {@code hostname}.
    * @throws Exception
    */
-  StatsWrapper queryStatsOf(String clusterName, String hostname) throws Exception;
+  StatsWrapper queryAccountStatsOf(String hostname) throws Exception;
```

Comment:
Name ending in "Of" is a bit weird.  Suggest queryServerAccountStats, or queryAccountStatsByHost().